module/VuFind/src/VuFind/RecordDriver/DefaultRecord.php

```diff
@@ -1722,4 +1722,80 @@ public function getCoordinateLabels()
         return isset($this->fields['long_lat_label'])
             ? $this->fields['long_lat_label'] : [];
     }
+
+    /**
+     * Get the bookplate names.
+     *
+     * @param $s string name of data to retrieve.
+     *
+     * @return array
+     */
+    public function getBookplateSolrData($s)
+    {
+        $data = [
+            'titles' => $this->getBookplateSolrTitlesField(),
+            'imgNames' => $this->getBookplateSolrImgNamesField()
+        ];
+        $field = $data[$s];
+        if (!empty($field) && isset($this->fields[$field])) {
+            return $this->fields[$field];
+        }
+        return [];
+    }
+
+    /**
+     * Get the full bookplate URL string template.
+     *
+     * @return string
+     */
+    public function getBookplateFullUrlTemplate()
+    {
+        return isset($this->recordConfig->Record->bookplate_full) ?
```

Comment:
Rather than hard-coding the configuration path to these settings, I think it would be better to leverage the configuration options of the "Related" module. See my comment on the constructor for more details.